app/Models/NewsFeed.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use DB;
+
+class NewsFeed extends Model
+{
+    protected $table = 'news_feed';
+    public $timestamps = false;
```

Comment:
We are using it because we are inserting data into database table and table has no created_at or updated_at columns so we have to indicate it in model.
we are using it in other models as well. you can see algorithm and ads model for reference.

Reply:
If I am not wrong then `$timestamps` creates `created_at` and `updated_at` with DateTime or TimeStamp type. But we using unix **timestamps**. So we have to be consistent.